docs/src/pages/system/the-sx-prop/the-sx-prop.md

```diff
@@ -124,7 +124,7 @@ _Head to the [sizing page](/system/sizing/) for more details._
 
 ### Spacing
 
-The spacing properties: `margin`, `padding` and the corresponding longhand properties multiply the values they receive by the `theme.spacing` value (the default for the value is `8px`).
+The spacing properties: `margin`, `padding` and their corresponding shorthand properties multiply the values they receive by the `theme.spacing` value (the default for the value is `8px`).
```

Comment:
Self-code review 😅 : changed from `longhand` to `shorthand` because in my head the sentence makes reference to the other props like `m`, `mt`, `mr`, `p`, etc from the below table, which are the shorthand properties. Which of course can be wrong ...

Reply:
I guess the `longhand` means `marginTop`, `marginBottom`, ...

Reply:
> I guess the `longhand` means `marginTop`, `marginBottom`, ...

That is exactly the other thing that crossed my mind and why I self review commented in the first place. Ok, let me revert that